test/jdk/java/security/cert/CertPathValidator/OCSP/GetAndPostTests.java

```diff
@@ -169,6 +161,20 @@ public static void main(String args[]) throws Exception {
         }
     }
 
+    private static void tryToStartOcspServer(SimpleOCSPServer ocspResponder) throws InterruptedException, IOException {
+        for (int i = 0 ; i < 6 ; ++i) {
```

Comment:
replace the iterations upper bound with a constant

private static final int MAX_SERVER_START_RETRY = 6;

or whatever you prefer as its moniker